### PR TITLE
Feat shed balances

### PR DESCRIPTION
**Pull request by zgfzgf**
use ActorNameByCode func
add miner-info flag

### CODE REVIEW

File: cmd/lotus-shed/balances.go [hunk 1]
```diff
@@ -125,6 +127,8 @@ var chainBalanceCmd = &cli.Command{
 				}
 				ai.Worker = info.Worker
 				ai.Owner = info.Owner
+				fmt.Printf("miner:%s, owner:%s, worker:%s, power:%s(%s)\n", ai.Address, ai.Owner,
```
Comment (magik6k) [inline cmd/lotus-shed/balances.go:130]: This command produces a CSV which is used in a number of places. This print breaks that output

File: cmd/lotus-shed/balances.go [hunk 2]
```diff
@@ -32,7 +22,13 @@ import (
 	"github.com/filecoin-project/go-state-types/big"
 
 	"github.com/filecoin-project/lotus/chain/actors/adt"
+	"github.com/filecoin-project/lotus/chain/actors/builtin"
+	ainit "github.com/filecoin-project/lotus/chain/actors/builtin/init"
 	"github.com/filecoin-project/lotus/chain/actors/builtin/miner"
+	"github.com/filecoin-project/lotus/chain/actors/builtin/multisig"
+	"github.com/filecoin-project/lotus/chain/actors/builtin/power"
+	"github.com/filecoin-project/lotus/chain/actors/builtin/reward"
+	"github.com/filecoin-project/lotus/chain/gen/genesis"
```
Comment (magik6k) [inline cmd/lotus-shed/balances.go:31]: Please don't commit changes to import order. Even if it's cleaner, it's annoying to review

**Review by magik6k — CHANGES_REQUESTED** [review]

File: cmd/lotus-shed/balances.go (hunk 1 above)
Comment (zgfzgf) [reply cmd/lotus-shed/balances.go:130]: add miner-info flag, default:It won't print

File: cmd/lotus-shed/balances.go (hunk 2 above)
Comment (zgfzgf) [reply cmd/lotus-shed/balances.go:31]: OK

**Comment by ZenGround0** [comment]
Hey @zgfzgf I'm going to close this down.  The main reason being that `./lotus-shed audits stateroot-balances --miner-info` does what it looks like you want already.

While I agree that `builtin.ActorNameByCode` is nicer than `string(act.Code.Hash()[2:])` they do have the same behavior so I'm leaving as is for now.  If you are motivated feel free to open a new PR with only the substitution of `builtin.ActorNameByCode`, ideally everywhere `string(act.Code.Hash()[2:])` is used in the codebase so we can clean these up all at once.

**Pull request closed without merging**